graphqlapi/src/test/java/io/stargate/graphql/schema/DmlSchemaCollectionsTest.java

```diff
@@ -10,7 +10,10 @@
 import io.stargate.db.schema.Keyspace;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.parallel.Execution;
+import org.junit.jupiter.api.parallel.ExecutionMode;
 
+@Execution(ExecutionMode.CONCURRENT)
```

Comment:
Would it be ok to put this annotation on base classes and use `@Isolated` on classes that we know cannot run in parallel with other tests?

Reply:
I tried that and it didn't seem to pick it up

Reply:
odd... it's marked as `@Inherited` :thinking: